src/vision.cs

```diff
@@ -144,6 +145,7 @@ enum VNDetectFaceLandmarksRequestRevision : ulong {
 		Unspecified = 0,
 		One = 1,
 		Two = 2,
+		Three = 3,
```

Comment:
Why isn't this getting an availability attribute?

Reply:
Nice catch